lib/Ix/DBIC/ResultSet.pm

```diff
@@ -411,10 +411,16 @@ sub ix_create ($self, $ctx, $to_create) {
     } else {
       my $exception = $@;
 
-      my $error;
+      my ($error, $res);
```

Comment:
I'm pretty sure that convention for this kind of return is always `($result, $error)`, not `($error, $result)`.  Even outside of Go.  :-)  Please follow suit.